### ISSUE TITLE
Screen readers NVDA and Narrator does not announce sort feature in column headers

### ISSUE THREAD

**Issue by angelbello87** [ag-grid/ag-charts#1329]
**I'm submitting a ...** (check one with "x")

```
[X] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Customers of AG Grid**

If you are a customer you are entitled to use the AG Grid's customer support system (powered by Zendesk). Please use that channel for guaranteed response from the AG Charts team with regards bugs, feature requests and support.

**Requesting Community Support**

If you are not a customer of AG Grid, our staff will label your issue as managed-by-the-community. This means that AG Grid staff will not be actively looking into it and it will be closed if inactive for more than one month. The community is welcome to help with this question/support issue.

**Providing a Reproducible Scenario**
Accepted reproducible scenarios are
1- Install a screen reader (NVDA or Narrator)
2- Open screen reader software
3- While enable, navegate to ag-grid component
4- Navegate with keyboard to a column header that has sort feature
5- Screen readers DO NOT announce sort feature

Current behavior:
Screen readers do not announce sort feature

Expected behavior: 
Screen readers should announce column sort feature

Note: Screen readers do announce sort after sorted is activated but needs to be announce as a feature of the column as well

![image](https://github.com/ag-grid/ag-charts/assets/14942214/b8af0290-c548-4c3b-90b6-a2eb119eb24b)

**Please tell us about your environment:**

Operating System: Windows 11 
Browser: Edge v124.0.2464.2 (64-bit)
AG-Grid: "@ag-grid-enterprise/all-modules": "27.3.0"
Angular: 13
TypeScript: 4.4.3

**Comment by AG-Zoheil** [comment]
Hello 

Thank you for the suggestion.

Currently there's no way to implement this behaviour with AG Grid. However, we agree this would be a valuable addition to the product and we have added a feature request for this.

We have added this requirement to our backlog and we are tracking it with the following reference: **AG-6510**
We have no timeline for this at the moment, but this can still be picked up by our team when they're making modifications to this functional area of AG Grid. Please use the functionality currently provided by AG Grid until this is delivered at a later point.

See whether this item will be in the next release by checking the NEXT RELEASE checkbox on the product pipeline page:
https://www.ag-grid.com/ag-grid-pipeline/

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Kind regards,
Zoheil
